spec/confirmation_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'confirmation'
+
+describe Confirmation do
+
+  subject(:confirmation) { described_class.new }
+
+  before do
+    allow(confirmation).to receive(:send_text) {"Your order has been placed and will be delivered around 10:00pm"}
+  end
+
+  # it 'should show the time an hour ahead in hours and minutes' do
+  #   expect(confirmation.time).not_to eq(Time.now)
```

Comment:
Try having a read of the testing chapter in POODR.  It talks about ways to test APIs by testing that the correct messages get sent to the API library with the correct arguments. eg check that `create` gets called on client.messages with the right message body.